lib/player_one.rb

```diff
@@ -0,0 +1,7 @@
+module TicTacToe
+    class FirstPlayer
+        attr_reader :choice
+        def make_a_move
+            @choice ="x"
+        end
```

Comment:
Beautifully simple, just the way I like it! 

But don't get too attached to it, I foresee it changing. Keep it this clean though. This is pretty work!